source/includes/azure/_clusters.md

```diff
@@ -148,3 +148,17 @@ Required | &nbsp;
 Optional | &nbsp;
 ------- | -----------
 `rootUsername`<br/>*string* | The user name to create a root user on cluster.
+
+<!-------------------- DELETE A CLUSTER -------------------->
```

Comment:
could you add the response example. i.e. 
{
 "taskStatus": "PENDING",
 "taskId": "123"
}

Reply:
016cc99